container_pipeline/lib/command.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+import subprocess
+
+
+def run(cmd):
+    """
+    Runs a shell command.
+
+    :param cmd: Command to run
+    :type cmd: str
+
+    :return: Command output
+    :rtype: str
+    :raises: subprocess.CalledProcessError
+    """
+    return subprocess.check_output(cmd, shell=True)
```

Comment:
1. I'd prefer handing subprocess errors from ``run_cmd`` in the calling function, as the needs of the callers might vary. The callers are usually lib helpers we've written, which promise to abstract out the complexities for their users. But, the lib helpers need to know what has happened.

1. ``subprocess.check_output`` blows up if the exit code from the called command is non zero. So, we won't get any output if the command has error'd out. If some command errors out without a non zero exit code (which should not be the case), the caller can handle it.